test/tests/mesh/add_sideset_ids/tests

```diff
@@ -31,4 +31,20 @@
     design = MooseMesh.md
     requirement = 'The system shall be able to remove sides from sidesets when tese sides elements are moved to new blocks.'
   []
+  [map_sidesets]
+    type = CSVDiff
+    input = simple.i
+    csvdiff = simple_out.csv
```

Comment:
this CSV should include surface areas not just volumes

Reply:
and number of nodes in the nodesets

Reply:
> and number of nodes in the nodesets

I tried using NodalSum, but the way the sideset works, the sideset doesn't exist on the mesh until after the `SidesetAroundSubdomainUpdater` switches it with the mask.

Reply:
Resolved